index_matroska.md

```diff
@@ -64,7 +64,7 @@ Matroska is an open standards project. This means for personal use it is absolut
 
 # Status of this document
 
-This document is a work-in-progress specification defining the Matroska file format as part of the [IETF Cellar working group](https://datatracker.ietf.org/wg/cellar/charter/). But since it's quite complete it is used as a reference for the development of libmatroska. Legacy versions of the specification can be found [here](https://matroska.org/files/matroska.pdf) (PDF doc by Alexander Noé -- outdated).
+This document is a work-in-progress specification defining the Matroska file format as part of the [IETF Cellar working group](https://datatracker.ietf.org/wg/cellar/charter/). But since it's quite complete it is used as a reference for the development of libmatroska.
```

Comment:
I'd like to keep the file and the link somewhere.

Reply:
I can definitely add it back.

Reply:
The old specs PDF by Alexander Noé is up again, as is the link in this file. I've renamed the file to be bit a less vague about its contents.